bin/foreman-certs

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+require 'rubygems'
+require 'kafo'
+
+CONFIG_DIR = './foreman_certs/config/'.freeze
+LAST_SCENARIO_PATH = './config/last_scenario.yaml'.freeze
```

Comment:
these are unused, right?

Reply:
argh. kafo uses `defined?(CONFIG_DIR) && CONFIG_DIR)`. whyyyyyyyyyyyyyyyyyyyyy.

last scenario is unused tho

Reply:
https://github.com/theforeman/foreman-installer/pull/1014 is related :)